@xen-orchestra/web-core/lib/utils/to-array.utils.ts

```diff
@@ -0,0 +1,5 @@
+import type { MaybeArray } from '@core/types/utility.type'
+
+export function toArray<T>(value: MaybeArray<T>): T[] {
+  return Array.isArray(value) ? value : [value]
```

Comment:
Don't know if that's relevant here but I often need to handle `undefined` as an empty array in this kind of utils.

Reply:
Yeah, I like the idea 👍🏻

I'm adding it.